store/store.go

```diff
@@ -15,6 +15,7 @@ import (
 type StoreResult struct {
 	Data interface{}
 	Err  *model.AppError
+	NErr error
```

Comment:
Let's add a comment here to clarify the purpose of this field.

```
// NErr a temporary field used by the new code for the AppError migration. This will later become Err when the entire store is migrated.
NErr error
```

Reply:
No problem! :+1: